home-exercises/src/main/java/advancedtest/exercise2/Basket.java

```diff
@@ -0,0 +1,42 @@
+package advancedtest.exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+    private T items;
+    private int currentCount = 0;
+    List<T> itemList = new ArrayList<>();
```

Comment:
Šim jābūt privātam, savādāk to varēs rediģēt no ārpuses un currentCount protams tad nemainīsies.